exonum/src/explorer/mod.rs

```diff
@@ -553,6 +553,18 @@ impl<'a> BlockchainExplorer<'a> {
         Some(TransactionInfo::Committed(tx))
     }
 
+    /// Return status of call in block
+    pub fn call_status(
+        &self,
+        block_height: Height,
+        call_location: &CallInBlock,
```

Comment:
Nit: `CallInBlock` is a `Copy` type (and quite small in size); it could make sense to pass it by value.